LoopKit/TherapySettings.swift

```diff
@@ -30,6 +44,9 @@ public struct TherapySettings: Equatable, Codable {
     
     public var insulinModel: StoredSettings.InsulinModel?
     
+    public var glucoseUnit: BGUnit?
```

Comment:
I'm a bit concerned that this duplicates the information in `glucoseTargetRangeSchedule?.unit`.  Is it necessary as a separate field, or can it be a computed property?

Reply:
Definitely, it can be computed. I made the change based on our discussion surrounding BG units for the settings overview screen, but am definitely not tied to it

Reply:
I'm definitely not opposed to having the `var`, I just think making it computed will avoid any possible mismatch.